dbapi/src/main/java/com/griddynamics/jagger/dbapi/provider/SessionInfoProviderImpl.java

```diff
@@ -84,8 +87,19 @@ public void saveUserComment(Long sessionDataId, String userComment) throws Runti
         dataSaverService.saveUserComment(sessionDataId, userComment);
     }
 
-    public Long getTotalSize() throws RuntimeException {
-        return (Long) entityManager.createQuery("select count(sessionData.id) from SessionData as sessionData").getSingleResult();
+    public Long getTotalSize() {
+        Long result;
+        try {
+            result = (Long) entityManager.createQuery("select count(sessionData.id) from SessionData as sessionData").getSingleResult();
+        } catch (PersistenceException ex) {
+            Throwable rootCause = ExceptionUtils.getRootCause(ex);
+            if (rootCause != null && StringUtils.contains(rootCause.getMessage(), "doesn't exist")) {
+                result = 0L;
```

Comment:
it doesn't look robust. Can you check by exception type? Because error message can be easily changed